js/testapps/rag/src/pdf-rag.ts

```diff
@@ -117,7 +117,7 @@ export const synthesizeQuestions = ai.defineFlow(
     const questions: string[] = [];
     for (let i = 0; i < chunks.length; i++) {
       const qResponse = await ai.generate({
-        model: gemini15Flash,
+        model: googleAI.model('gemini-2.5-flash'),
```

Comment:
I can't tell if it's intentional switch from vertex to googleai or copy/paste oversight.